### DISCUSSION TITLE
LFS openrc compile without pam

### DISCUSSION THREAD

**Discussion by Ardaninho** [OpenRC/openrc#571] [Q&A] [answered]
Hello, i was wondering if I could compile OpenRC without the need of pam.

Linux From Scratch uses SysVinit as their initsystem but I dont want that.
I was having issues that pam was not found because LFS did not include pam.

If there's a way that I can compile OpenRC without the need of pam, I would appreciate.
Thanks.

**Comment by vapier** [comment, marked as answer]
yes, this has always worked.  use `meson configure` to see available options.
